integration/testrunner/grpc.go

```diff
@@ -308,28 +347,66 @@ func createConnection(ctx context.Context, t *testing.T, addr, idToken string) *
 // We specifically look up the log using the UUID specified in the request as we know the server will add that
 // as the resource name, and provides us a unique key to find logs with.
 func (g *GRPC) makeQueryForGRPCUnary(id string) *bigquery.Query {
-	// Cast to int64 because the result checker expects a number.
-	queryString := fmt.Sprintf("SELECT CAST(EXISTS (SELECT * FROM `%s.%s` WHERE jsonPayload.resource_name=?", g.ProjectID, g.DatasetQuery)
-	if g.RequireJustification {
-		// TODO(#265): For back-compatibility, we need to ensure justification in either metadata key.
-		queryString += ` AND jsonPayload.metadata.justification IS NOT NULL`
-	}
-	queryString += ` AND jsonPayload.service_name IS NOT NULL`
-	queryString += ` AND jsonPayload.method_name IS NOT NULL`
-	queryString += ` AND jsonPayload.authentication_info.principal_email IS NOT NULL`
-	queryString += ") AS INT64)"
+	queryString := fmt.Sprintf(`SELECT
+		jsonPayload.method_name AS MethodName,
```

Comment:
No need to select these fields since we want to get the result and convert it to Logentry(https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry)